config/test/core.js

```diff
@@ -0,0 +1,29 @@
+import { resolve } from 'path';
+import snapshotResolver from './snapshotResolver';
+
+export default {
+  test: {
+    alias: [
+      {
+        find: /\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|ico)$/,
+        replacement: '',
+        customResolver() {
+          return resolve(__dirname, '../mocks/fileMock.js')
+        }
+      }],
+    name: 'core',
+    globals: true,
```

Comment:
I saw that we are importing vitest functions like `describe`, `expect`, `it`, `beforeAll`, etc.

But, adding this `globals: true` 
allows us to write our tests/config files without the need to import those functions.

What do you think about removing the `import` from all test files? I can do a commit in this branch to remove all of them, and the typescript configuration to work with it properly too.

Reply:
I don't particularly appreciate using globals; the more we can avoid, the better. 

Besides my personal opinion on that, the main reason for this config to be here is that `jest-styled-components` use globals, `global.beforeEach` to be more precise, and we need this config for this package to work. 

See: https://github.com/styled-components/jest-styled-components/issues/431

So combining `globals: true` + `tsconfig` give us a guarantee that `jest-styled-components` works and avoids globals inside tests which is what I was aiming for with these changes.

Does it make sense to you?

Reply:
Alright, totally agree!

Maybe in the future, we can update our `CONTRIBUTING.md` or add some Eslint rules to force those imports. Just to avoid missing those imports.